train.py

```diff
@@ -110,7 +110,8 @@ def collect_files(self):
         self.multi_speaker = len(l) == 5
         texts = list(map(lambda l: l.decode("utf-8").split("|")[3], lines))
         if self.multi_speaker:
-            speaker_ids = list(map(lambda l: int(l.decode("utf-8").split("|")[-1]), lines))
+            speaker_ids = list(
```

Comment:
Could you revert unrelated changes?

Reply:
yep. sorry, my editor auto-fixes too long lines